content/v2.0/account-management/pricing-plans.md

```diff
@@ -46,15 +46,24 @@ Data retention is determined by the time at which data is written to InfluxDB; n
 
 _To remove rate limits, [upgrade to a Usage-based Plan](/v2.0/account-management/billing/#upgrade-to-usage-based-plan)._
 
-## Usage-based Plan
+## Usage-Based Plan
 
-The Usage-based Plan offers more flexibility and ensures you only pay for what you [use](/v2.0/account-management/data-usage/).
+The Usage-Based Plan offers more flexibility and ensures you only pay for what you [use](/v2.0/account-management/data-usage/).
 
-#### Usage-based Plan rate limits
+### Pricing vectors
 
-To protect against any intentional or unintentional harm, Usage-based Plans include soft rate limits:
+The following two vectors determine pricing:
 
-- **Writes:** 300MB every 5 minutes
+- **Data out** is the total sum of the data (measured in GB) returned to the user to answer a query, also known as data transfer costs.
+- **Query count** is the total number of individual query operations. Each individual operation — including
+queries, tasks, alert checks, notifications, and Data Explorer activity — is one billable query operation. Refreshing a dashboard with multiple cells counts as multiple query operations. Failed query or task executions do not add to your query count.
```

Comment:
Should we explicitly state that each dashboard cell will count as 1 query execution/operation on refresh?